### ISSUE TITLE
handle symbol collision

### ISSUE THREAD

**Issue by shaunlebron** [shaunlebron/cljs-api-parser#5]
sometimes we have function wrappers over macros of the same name.

- make sure this is true for all cases
- just prefer the function and remove the macro

**Comment by shaunlebron** [comment]
all symbols with collisions:

```
cljs.core_-.cljsdoc
cljs.core_EQEQ.cljsdoc
cljs.core_GT.cljsdoc
cljs.core_GTEQ.cljsdoc
cljs.core_LT.cljsdoc
cljs.core_LTEQ.cljsdoc
cljs.core_PLUS.cljsdoc
cljs.core_SLASH.cljsdoc
cljs.core_STAR.cljsdoc
cljs.core_aget.cljsdoc
cljs.core_alength.cljsdoc
cljs.core_array-map.cljsdoc
cljs.core_array.cljsdoc
cljs.core_aset.cljsdoc
cljs.core_assert.cljsdoc
cljs.core_bit-and-not.cljsdoc
cljs.core_bit-and.cljsdoc
cljs.core_bit-clear.cljsdoc
cljs.core_bit-flip.cljsdoc
cljs.core_bit-not.cljsdoc
cljs.core_bit-or.cljsdoc
cljs.core_bit-set.cljsdoc
cljs.core_bit-shift-left.cljsdoc
cljs.core_bit-shift-right-zero-fill.cljsdoc
cljs.core_bit-shift-right.cljsdoc
cljs.core_bit-test.cljsdoc
cljs.core_bit-xor.cljsdoc
cljs.core_byte.cljsdoc
cljs.core_dec.cljsdoc
cljs.core_double.cljsdoc
cljs.core_falseQMARK.cljsdoc
cljs.core_float.cljsdoc
cljs.core_for.cljsdoc
cljs.core_hash-map.cljsdoc
cljs.core_hash-set.cljsdoc
cljs.core_identicalQMARK.cljsdoc
cljs.core_inc.cljsdoc
cljs.core_instanceQMARK.cljsdoc
cljs.core_int.cljsdoc
cljs.core_js-delete.cljsdoc
cljs.core_js-mod.cljsdoc
cljs.core_js-obj.cljsdoc
cljs.core_keywordQMARK.cljsdoc
cljs.core_list.cljsdoc
cljs.core_make-array.cljsdoc
cljs.core_max.cljsdoc
cljs.core_min.cljsdoc
cljs.core_negQMARK.cljsdoc
cljs.core_nilQMARK.cljsdoc
cljs.core_numberQMARK.cljsdoc
cljs.core_posQMARK.cljsdoc
cljs.core_short.cljsdoc
cljs.core_str.cljsdoc
cljs.core_stringQMARK.cljsdoc
cljs.core_symbolQMARK.cljsdoc
cljs.core_trueQMARK.cljsdoc
cljs.core_unchecked-add-int.cljsdoc
cljs.core_unchecked-add.cljsdoc
cljs.core_unchecked-byte.cljsdoc
cljs.core_unchecked-char.cljsdoc
cljs.core_unchecked-dec-int.cljsdoc
cljs.core_unchecked-dec.cljsdoc
cljs.core_unchecked-divide-int.cljsdoc
cljs.core_unchecked-double.cljsdoc
cljs.core_unchecked-float.cljsdoc
cljs.core_unchecked-inc-int.cljsdoc
cljs.core_unchecked-inc.cljsdoc
cljs.core_unchecked-multiply-int.cljsdoc
cljs.core_unchecked-multiply.cljsdoc
cljs.core_unchecked-negate-int.cljsdoc
cljs.core_unchecked-negate.cljsdoc
cljs.core_unchecked-remainder-int.cljsdoc
cljs.core_unchecked-short.cljsdoc
cljs.core_unchecked-subtract-int.cljsdoc
cljs.core_unchecked-subtract.cljsdoc
cljs.core_undefinedQMARK.cljsdoc
cljs.core_unsigned-bit-shift-right.cljsdoc
cljs.core_vector.cljsdoc
cljs.core_zeroQMARK.cljsdoc
```

**Comment by shaunlebron** [comment]
because of the order at which we are parsing symbols, naturally the cljs forms will overwrite the clj forms, and the forms further down the file will replace previous forms of the same name.  This should cover the cases.

To fix, I just got rid of the `:append true`: 68615c96c036ed138c1360975cfe12637c9b6b6e
